dandi/download.py

```diff
@@ -177,17 +263,75 @@ def follow_redirect(url):
 follow_redirect = _dandi_url_parser.follow_redirect
 
 
-def download(
+def download(urls, output_dir, *, format="pyout", existing="error", jobs=1):
+    # TODO: unduplicate with upload. For now stole from that one
+    # We will again use pyout to provide a neat table summarizing our progress
+    # with upload etc
+    import pyout
+    from .support import pyout as pyouts
+
+    # dandi.cli.formatters are used in cmd_ls to provide switchable
+    pyout_style = pyouts.get_style(hide_if_missing=False)
+
+    rec_fields = ("path", "size", "done", "done%", "checksum", "status", "message")
```

Comment:
If we're going to be passing tuples of `columns` to pyout, we should add `!= 0.6.0` to the dependency in `setup.cfg` in order to avoid the version that doesn't support them.

Reply:
good idea! you did fix for 0.6.0 for upload in cc4d37cda83584c69778e6fcecebc9124dd71ed8  but there is still `ls`, and I just feel it would be just easier to ask to avoid 0.6.0. done in 096c8dd232b665eb6a15aaeb7cf9b9b3aee95d87